Makefile

```diff
@@ -175,7 +184,8 @@ lint-go:
 	hack/lint-go.sh
 
 lint-admin-portal:
-	docker build -f Dockerfile.portal_lint . -t linter
+	echo $(REGISTRY)
```

Comment:
reminder to remove this debugging statement

Reply:
Good catch, thanks!